### PR TITLE
Category management

### PR DESCRIPTION
**Pull request by IvanZabor**
Create endpoints to manage Categories with CRUD

### CODE REVIEW

**Review by kosenkoNick — APPROVED** [review]

**Review by VKoniukh — APPROVED** [review]
Everything is Ok!

File: src/main/java/com/itacademy/cms/mapper/CategoryMapper.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package com.itacademy.cms.mapper;
+
+import com.itacademy.cms.model.Category;
+import com.itacademy.cms.model.dto.CategoryDto;
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+
+@Mapper(componentModel = "spring")
+public interface CategoryMapper {
```
Comment (dmakhun) [inline src/main/java/com/itacademy/cms/mapper/CategoryMapper.java:9]: please use one "MapStructMapper" config interface for all mappings

File: src/test/java/com/itacademy/cms/CategoryTest.java [hunk 2]
```diff
@@ -0,0 +1,138 @@
+//package com.itacademy.cms;
```
Comment (dmakhun) [inline src/test/java/com/itacademy/cms/CategoryTest.java:1]: please provide valid working tests
do not commit commented code

**Review by dmakhun — DISMISSED** [review]

**Comment by dmakhun** [comment]
@Kv-Java-Intensive/the-team 

please use Uuid instead of DB Id in endpoints -> find an Entity by Uuid, then do necessary manipulations
uuid should be added to an appropriate Entity

File: src/main/java/com/itacademy/cms/mapper/CategoryMapper.java (hunk 1 above)
Comment (IvanZabor) [reply src/main/java/com/itacademy/cms/mapper/CategoryMapper.java]: Created one "MapStructMapper" config interface for all mappings

File: src/test/java/com/itacademy/cms/CategoryTest.java (hunk 2 above)
Comment (IvanZabor) [reply src/test/java/com/itacademy/cms/CategoryTest.java:1]: Add @Ignore for CategoryTest test-methods

**Pull request merged**